rebound/particle.py

```diff
@@ -119,7 +119,7 @@ def __init__(self, particle=None, m=None, x=None, y=None, z=None, vx=None, vy=No
         orbi = [primary,a,e,inc,Omega,omega,pomega,f,M,l,theta]
         
         self.m = 0. if m is None else m
-        self.id = -1 if id is None else id
```

Comment:
By the way, I personally find this syntax a little hard to read. The following seems more straight forward:

```python
if m is None:
    self.m = 0.
else:
    self.m = m
```